JRC Publications Repository.js

```diff
@@ -67,10 +67,21 @@ function getSearchResults(doc, checkOnly) {
 function detectWeb(doc, url) {
 	var type = ZU.xpath(doc, '//meta[@name="DC.type"]/@content');
 	if (type.length>0) {
-		if (mappingTable[type[0].value]) {
-			return mappingTable[type[0].value];
-		} else {//generic fallback
-			Z.debug('Unrecognized item type: ' + type[0].value);
+		if (mappingTable[type[0].textContent]) {
+			return mappingTable[type[0].textContent];
```

Comment:
Also, you could do what some of the code does below and retrieve the node with the XPath, then grab the attribute via object property (`.content`) or `.getAttribute` if the property is not available. I think this would be easier to keep track of and it sometimes performs more appropriately (e.g. `.href`)